integrations/drip/drip_test.go

```diff
@@ -0,0 +1,177 @@
+package drip
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/jipiboily/forwardlytics/integrations"
+)
+
+func TestIdentifyErrorWhenNoEmail(t *testing.T) {
+	drip := Drip{}
+	identification := integrations.Identification{
+		UserID:     "123",
+		UserTraits: map[string]interface{}{},
+	}
+	err := drip.Identify(identification)
+	if err == nil {
```

Comment:
I know this is how I built the `intercom` package but now I would go one step further and make sure that I receive the proper error message.

You don't have to do it, just thought it was worth mentioning 😀